hkmc2/shared/src/main/scala/hkmc2/semantics/Symbol.scala

```diff
@@ -79,6 +79,13 @@ class TempSymbol(val trm: Opt[Term], dbgNme: Str = "tmp")(using State) extends B
   override def toLoc: Option[Loc] = trm.flatMap(_.toLoc)
   override def toString: Str = s"$$${super.toString}"
 
+
+class RefSymbol(val origin: Symbol)(using State) extends LocalSymbol:
```

Comment:
What's the purpose of this class? Either document that purpose with a small comment or remove the class.

Reply:
Sorry, this is missing in the previous commit. I've fixed it in the following one.